pkg/pillar/cmd/zedkube/applogs.go

```diff
@@ -16,32 +17,37 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/kubeapi"
 	"github.com/lf-edge/eve/pkg/pillar/types"
+	uuid "github.com/satori/go.uuid"
 	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/client-go/kubernetes"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// collectAppLogs - collect App logs from pods which covers both containers and virt-launcher pods
```

Comment:
Is the function comment removed because it is outdated? I suggest to update the comment instead.